scene/theme/default_theme.cpp

```diff
@@ -31,7 +31,9 @@
 #include "default_theme.h"
 
 #include "core/io/image.h"
+#ifdef BROTLI_ENABLED
 #include "default_font.gen.h"
+#endif
 #include "default_theme_icons.gen.h"
```

Comment:
While at it, these should use `scene/theme/` absolute paths.

And I'd move the `#ifdef BROTLI_ENABLED` block separate from the main block of includes so it doesn't break alphabetical ordering from clang-format.